android/quest/src/main/java/org/smartregister/fhircore/quest/ui/questionnaire/QuestionnaireViewModel.kt

```diff
@@ -113,6 +110,11 @@ constructor(
     )
   }
 
+  // TODO this setting should be loaded from a configuration
+  var isDeactivateMembers = false
```

Comment:
can you describe what this var controls? From the name that is not unclear to me